Source/ARTRealtimeChannel.m

```diff
@@ -741,11 +741,13 @@ - (void)setAttached:(ARTProtocolMessage *)message {
     }
 
     if (self.state_nosync == ARTRealtimeChannelAttached) {
-        if (message.error != nil) {
-            _errorReason = message.error;
+        if (!message.resumed) { // RTL12
```

Comment:
Where is this new behaviour tested? It's not clear to me.

Reply:
https://github.com/ably/ably-cocoa/pull/1791/commits/18f5f6ca27f47eaccb234d57a3a6350e03e504f6